### ISSUE TITLE
Convert to CoreData, where appropriate

### ISSUE THREAD

**Issue by sethgagnon** [Artificial-Pancreas/iAPS#367]
@Jon-b-m 
Need to add some detail here as to what is remaining for CoreData conversion work

**Comment by Jon-b-m** [comment]
* Suggestion (all data computed in the final determine_basaal.js is outputted to suggestion.json. Enacted bolus, predictions etc. This should be saved to CoreData instead of to a JSON for us to use it everywhere (like in a new chart to see history of every loop). In oref this data is returned as rT… for instance rT.iob will be suggestion.iob in iAPS. This is the most complex model to convert to CoreData and the one most needed right now.  

* basals, CR, ISF, targets JSON

* pump history (insulin enacted)

* Meals 

* autotune 

* auotsens 

* IOB (is retrieved from suggestion. create a relation. 

* COB (is retrieved from suggestion. Create a relation)

* basically every other JSON also, except glucose which is already done (this one just needs to be used everywhere, but it’s in the pipeline)
